gateway-service-baseline-lib/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    `java-library`
+    jacoco
+    id("org.hypertrace.jacoco-report-plugin")
+    id("org.hypertrace.publish-plugin")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+dependencies {
+    api(project(":gateway-service-api"))
```

Comment:
It's a utility lib, so we have to remove the deps on this lib. For now, can you expose the method `private static Baseline getBaseline(double[] metricValueArray)` as public. As I guess, `hypetrace-alert-engine` doesn't understand gateway service `Value` proto.

Reply:
I just saw that hypertrace-alert-engine is already using gateway-service-api.